plugins/optimization-detective/dashboard.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Optimizing for Optimization Detective.
+ *
+ * @package optimization-detective
+ * @since 0.1.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+function od_add_submenu_page() {
+	add_submenu_page(
+		'index.php',
+		__( 'Performance', 'optimization-detective' ),
+		__( 'Performance', 'optimization-detective' ),
+		'manage_options',
+		'od-performance-dashboard',
+		'od_render_dashboard_page'
+	);
+}
+
+add_action( 'admin_menu', 'od_add_submenu_page' );
+
+function od_render_dashboard_page() {
+	$asset_file = plugin_dir_path( __FILE__ ) . 'build/performance-dashboard.asset.php';
+
+	if ( ! file_exists( $asset_file ) ) {
+		return;
+	}
+
+	$asset = include $asset_file;
```

Comment:
Probably better to error if the file is absent, as it means a build needs to happen. In fact, `load.php` will raise an error if a build hasn't happened.

```suggestion
	$asset = require $asset_file;
```